deploy/charts/rook-ceph/values.yaml

```diff
@@ -305,22 +308,22 @@ csi:
   # CEPH CSI NFS plugin resource requirement list, Put here list of resource
   # requests and limits you want to apply for plugin pod
   csiNFSPluginResource: |
-   - name : driver-registrar
```

Comment:
can you please revert this change?

Reply:
not sure, tried to do that manually, but on saving, it gets back to this, not sure why these changes occurred. Will check this.